components/02-elements/breadcrumbs/breadcrumbs.hbs

```diff
@@ -1,21 +1,26 @@
-<div class="breadcrumbs {{ class }}" {{{ attributes }}}>
+<ol
+    class="breadcrumbs {{ class }}" {{{ attributes }}}
+    aria-label="{{ ariaLabel }}"
+    role="navigation"
+>
     {{#each categories}}
         {{#if @last}}
-            <span class="breadcrumbs__item"
-                  aria-label="{{ ariaLabel }}"
-                  tabindex="0"
+            <li
+                class="breadcrumbs__item"
+                tabindex="0"
             >
                 {{ name }}
-            </span>
+            </li>
         {{else}}
-            <span class="breadcrumbs__item">
-                <a href="{{ href }}"
-                   class="{{ linkClass }}"
-                   aria-label="{{ ariaLabel }}"
+            <li class="breadcrumbs__item">
+                <a
+                    href="{{ href }}"
+                    class="{{ linkClass }}"
                 >
                     {{ name }}
                 </a>
-            </span>
+            </li>
         {{/if}}
     {{/each}}
 </div>
```

Comment:
change this closing tag to `</ol>`

Reply:
For the beginning it was for me a lot of code & empty link etc, but I've check it with screen reader in different browser and it work fine, check my codepen: https://codepen.io/anqa/pen/BPRMNw